numpy/core/tests/test_deprecations.py

```diff
@@ -741,6 +741,7 @@ def test_string(self):
         self.assert_deprecated(np.datetime64, args=('2000-01-01T00Z',))
 
     @dec.skipif(not _has_pytz, "The pytz module is not available.")
+    @dec.knownfailureif(sys.version_info[0:2] < (2, 7))
```

Comment:
Alright, I'll leave it then. I did make note of it in the commit message. If you thinking the wording needs tweaking, let me know.